ui/components/MesheryPatterns/MesheryPatternCard.js

```diff
@@ -0,0 +1,267 @@
+}) {
+
+  function genericClickHandler(ev, fn) {
+    ev.stopPropagation();
+    fn();
+  }
+  const [gridProps, setGridProps] = useState(INITIAL_GRID_SIZE);
+  const [fullScreen, setFullScreen] = useState(false);
+
+  const toggleFullScreen = () => {
+    setFullScreen(!fullScreen);
+  };
+
+  const classes=useStyles()
+
+  return (
+    <FlipCard
+      onClick={() => {
+        console.log(gridProps)
+        setGridProps(INITIAL_GRID_SIZE)
+        console.log("card clicked")
+      }}
+      duration={600}
+    >
+      {/* FRONT PART */}
+      <>
+        <div>
+          <Typography variant="h6" component="div">
+            {name}
+          </Typography>
+          <div className={classes.lastRunText} >
+            <div>
+              {updated_at
+                ? (
+                  <Typography color="primary" variant="caption" style={{ fontStyle : "italic" }}>
+                  Modified On: <Moment format="LLL">{updated_at}</Moment>
+                  </Typography>
+                )
+                : null}
+            </div>
+          </div>
+        </div>
+        <div className={classes.bottomPart} >
+
+          <div className={classes.cardButtons} >
+            <Button
+              variant="contained"
+              onClick={(ev) =>
+                genericClickHandler(ev, setSelectedPatterns)
+              }
+              className={classes.testsButton}
+            >
+              <Avatar src="/static/img/pattwhite.svg" className={classes.iconPatt} imgProps={{ height : "16px", width : "16px" }} />
+              Design
+            </Button>
+
+            <Button
+              variant="contained"
+              color="primary"
+              onClick={(ev) =>
+                genericClickHandler(ev, handleDeploy)
+              }
+            >
+              <TrendingUp className={classes.iconPatt}/>
+              Deploy
+            </Button>
+          </div>
+        </div>
+      </>
+
+      {/* BACK PART */}
+      <>
+        <Grid className={classes.backGrid}
+          container
+          spacing={1}
+          alignContent="space-between"
+          alignItems="center"
+        >
+          <Grid item xs={8} className={classes.yamlDialogTitle}>
+            <Typography variant="h6" className={classes.yamlDialogTitleText}>
+              {name}
+            </Typography>
+            <IconButton
+              onClick={(ev) =>
+                genericClickHandler(ev, () => {
+                  {
+                    toggleFullScreen()
+                    console.log("maximized Clicked")
+                  }
+                })
+              }
+              className={classes.maximizeButton}
+            >
+              {fullScreen ? <FullscreenExit /> : <Fullscreen />}
+            </IconButton>
+          </Grid>
+          <Grid item xs={12}
+            onClick={(ev) =>
+              genericClickHandler(ev, () => {
+                {
+                  console.log("yaml clicked")
```

Comment:
Same for this line 😶

Reply:
yes totally, will do that. PR is not ready to merge so there might be some `console.log` 😓 . Will do a proper cleanup once everything is implemented